globals.json

```diff
@@ -215,6 +221,7 @@
 		"Crypto": false,
 		"CryptoKey": false,
 		"CSS": false,
+		"CSSAnimation": false,
```

Comment:
Where do CSSAnimation and CSSTransition come from? They don't appear in the spec you referenced.